.github/workflows/test_deploy.yaml

```
@@ -0,0 +1,59 @@
+# Testing using KinD https://kind.sigs.k8s.io/docs/user/loadbalancer/
+name: Test the deployment of the load balancer
+
+on: pull_request
+
+jobs:
+  test-loadbalancer:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.head_ref }}
+      - name: Create k8s Kind Cluster
+        uses: helm/kind-action@v1.1.0
+      - name: Install loadbalancer
+        run: helm install -f test/values.yaml test-lb chart --namespace metallb-system --create-namespace
+      - name: Deploy nginx
+        run: kubectl apply -f test/nginx-test.yaml
+      - name: Sleep for 30 seconds
+        uses: juliangruber/sleep-action@v1
```

Wouldn't bash sleep work aswell and remove an external dependency?
```suggestion
        run: sleep 30
```